### PR TITLE
fix(swagger-ui): throw an error when "/" is set as swagger-ui path

### PR DESCRIPTION
**Pull request by machi1990**
The value blocks the application from serving anything else - see
https://github.com/quarkusio/quarkus/issues/3857#issuecomment-528149421

### CODE REVIEW

File: extensions/swagger-ui/deployment/src/main/java/io/quarkus/swaggerui/deployment/SwaggerUiProcessor.java [hunk 1]
```diff
@@ -181,6 +187,7 @@ public String updateApiUrl(String original) {
     static final class SwaggerUiConfig {
         /**
          * The path of the swagger-ui servlet.
+         * The value `/` is not allowed as it blocks the application from serving anything else.
```
Comment (gsmet) [inline extensions/swagger-ui/deployment/src/main/java/io/quarkus/swaggerui/deployment/SwaggerUiProcessor.java]: A `<p>` just before this sentence would be welcome.

**Review by gsmet — CHANGES_REQUESTED** [review]
Looks good apart from some very minor comment.

File: extensions/swagger-ui/deployment/src/main/java/io/quarkus/swaggerui/deployment/SwaggerUiProcessor.java [hunk 2]
```diff
@@ -181,6 +187,9 @@ public String updateApiUrl(String original) {
     static final class SwaggerUiConfig {
         /**
          * The path of the swagger-ui servlet.
+         * <p>
+         * The value `/` is not allowed as it blocks the application from serving anything else.
+         * </p>
```
Comment (gsmet) [inline extensions/swagger-ui/deployment/src/main/java/io/quarkus/swaggerui/deployment/SwaggerUiProcessor.java]: Ah ah, that's why I was precise about just adding a `<p>` before :). You don't need an ending p and Javadoc throws a warning for that.

Comment (machi1990) [reply extensions/swagger-ui/deployment/src/main/java/io/quarkus/swaggerui/deployment/SwaggerUiProcessor.java]: :-) Let me change that.

**Review by gsmet — APPROVED** [review]

**Pull request merged**